### ISSUE TITLE
get rid of NodeJS

### ISSUE THREAD

**Issue by mjakubicek** [elexis-eu/lexonomy#128]
Step 1 is to have a proxy in Python or Go that will allow transparent removal of parts of the backend without changes in the frontend.

**Comment by iiegn** [comment]
maybe, somewhere between 1. and 2., @michmech have the current API explicitly 'documented' or, at least, some core functionality of it: 
```
/ENDPOINT 
 - input: data (format + semantics)
 - returns: data (format + semantics)
```

This is somewhat selfish: i would like to use lexonomy within our neologism curation task. To do so, i would need to map the rather petty API [from here](https://gitlab.inf.unibz.it/commul/styrlogism/ws/blob/master/app/frontend.py) onto lexonomy. 

This is likely related to #61 (and could be considered a new task entirely - but know what is still missing, the API documentation would help...)

**Comment by mjakubicek** [comment]
Well, the current "official" API is documented here:

https://www.lexonomy.eu/push.api

Perhaps you could extend that one.

**Comment by michmech** [comment]
Yes, documenting Lexonomy's server-side endpoints (we can call that an "API") is desirable and doable. I'll be happy to do that.

No, this issue is not related to issue #61. Issue #61 is about the API Sketch Engine provides to Lexonomy for *pulling*. That's something else entirely.

Yes, this issue is related to issue #63. Issue #63 is about this API: https://www.lexonomy.eu/push.api which Lexonomy provides to Sketch Engine for *pushing*.

**Comment by iiegn** [comment]
> Yes, documenting Lexonomy's server-side endpoints (we can call that an "API") is desirable and doable. I'll be happy to do that.

just made a clickable sub-task for you ;)

> No, this issue is not related to issue #61. Issue #61 is about the API Sketch Engine provides to Lexonomy for _pulling_. That's something else entirely.
> 
> Yes, this issue is related to issue #63. Issue #63 is about this API: https://www.lexonomy.eu/push.api which Lexonomy provides to Sketch Engine for _pushing_.

i have been wondering about this distinction - but now i (think i) understand :)

**Comment by mjakubicek** [comment]
first attempt (proxy) in 9eaa7c946f1573db9a120761ec2ba5ae196dc2c8
